frontend/src/pages/admin/Template/sections/Catalogues.vue

```diff
@@ -0,0 +1,180 @@
+                </div>
+            </div>
+            <div class="flex flex-col sm:flex-row">
+                <FormRow v-model="input.url" class="w-full max-w-md sm:mr-8" :error="input.error" :disabled="readOnly">
+                    <template #append>
+                        <ff-button kind="secondary" size="small" :disabled="readOnly" @click="addURL()">
+                            <template #icon>
+                                <PlusSmIcon />
+                            </template>
+                        </ff-button>
+                    </template>
+                </FormRow>
+            </div>
+            <ff-button class="mt-2" size="small" :disabled="addDefaultEnabled" @click="addDefault()">
+                Add Default
+                <template #icon>
+                    <PlusSmIcon />
+                </template>
+            </ff-button>
+        </div>
+    </form>
+</template>
+
+<script>
+import { PlusSmIcon, XIcon } from '@heroicons/vue/outline'
+
+import SemVer from 'semver'
+
+import FormHeading from '../../../../components/FormHeading.vue'
+import FormRow from '../../../../components/FormRow.vue'
+import ChangeIndicator from '../components/ChangeIndicator.vue'
+import LockSetting from '../components/LockSetting.vue'
+
+export default {
+    name: 'TemplateCatalogueEditor',
+    components: {
+        FormRow,
+        FormHeading,
+        ChangeIndicator,
+        LockSetting,
+        PlusSmIcon,
+        XIcon
+    },
+    props: {
+        editTemplate: {
+            type: Boolean,
+            default: false
+        },
+        modelValue: {
+            type: Object,
+            required: true
+        },
+        readOnly: {
+            type: Boolean,
+            default: false
+        },
+        project: {
+            type: Object,
+            required: false,
+            default: null
+        }
+    },
+    emits: ['update:modelValue'],
+    data () {
+        return {
+            showAddRow: false,
+            input: {
+                url: '',
+                error: ''
+            },
+            urls: []
+        }
+    },
+    computed: {
+        editable: {
+            get () {
+                return this.modelValue
+            },
+            set (localValue) {
+                this.$emit('update:modelValue', localValue)
+            }
+        },
+        projectLauncherCompatible () {
+            if (this.editTemplate) {
+                // When editing template we don't have a project
+                return true
+            }
+
+            const launcherVersion = this.project?.meta?.versions?.launcher
+            if (!launcherVersion) {
+                // We won't have this for a suspended project - so err on the side
+                // of permissive
+                return true
+            }
+            // TODO needs to semver >= 1.12.0
+            return SemVer.satisfies(launcherVersion, '>=1.11.0')
```

Comment:
**TODO** this needs updating to match the release it actually ships in.

Reply:
This business logic being in the frontend logic feels slightly off. Not going to block, but it'd feel better if this threshold was grabbed from the server.